modules/environment-roles/root.tf

```diff
@@ -51,93 +51,18 @@ resource "aws_iam_role_policy_attachment" "shared_policy_attachment_1" {
 }
 
 resource "aws_iam_role_policy_attachment" "shared_policy_attachment_2" {
```

Comment:
Might want to rename this: `shared_iam_policy_attachment` for consistency.

Reply:
Ah yeah good point. I did a find and replace so I missed this one. I've changed the other one too.